gluster-exporter/metric_pv_lv_vg_test.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestMetric_PV_LV_VG(t *testing.T) {
```

Comment:
I feel this you needed for your local testing only. Please remove this.

Reply:
@shtripat , my 2cents will be to keep the testing files, as moving forward we will be checking functional correctness and it will be easier for others to do test first to check their setup is good or their changes has broken anything.
For example with the latest check-in, came to know (while testing) that there is a global object 'gluster' and we have to do a null check on the 'gluster' object first before using it.
As you have pointed out, the initial '*_test.go' file didn't have much, but will be adding lot of functional tests into it.

Reply:
As discussed later if we have some kind of unit test framework we will think of having these things done.